src/mixins/itext_behavior.mixin.js

```diff
@@ -172,7 +172,7 @@
 
       //  make sure we clear context even if instance is not editing
       if (shouldClear) {
-        var ctx = this._clearContextTop();
+        var ctx = this.clearContextTop(true);
```

Comment:
Ctx is restored on the next line so can we let the method restore it?

Reply:
because i did a blind search/replace 😄 
correcting this